src/data/languages/languageData.ts

```diff
@@ -34,6 +34,7 @@ export default {
   liveObjects: {
     javascript: '2.9',
     swift: '0.1',
+    java: '1.2.55',
```

Comment:
I think it's time for us to move to the 1.3

Reply:
done